src/Components/CalendarComponent/CalendarComponent.tsx

```diff
@@ -123,6 +123,7 @@ const CalendarComponent = (props: CalenderComponentProps) => {
       if (currentEvent) {
         return (
           <Week
+           startDate = {currentEvent.date}
```

Comment:
Before this was not being passed as prop causing the nothing happened here message to display.

Reply:
Good catch!

Reply:
Awesome catch, makes sense!